packages/AndroidManifest.template.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
We can remove the Android manifest, it won't make effect in final apps.

Reply:
Fine by me, I was just testing it properly